examples/crowd-funding/src/contract.rs

```diff
@@ -138,10 +210,11 @@ impl CrowdFunding<ViewStorageContext> {
 
     /// Checks that the sessions pledged all use the correct token.
     fn check_session_tokens(&self, sessions: &[SessionId]) -> Result<(), Error> {
+        let fungible_id = Self::fungible_id()?;
```

Comment:
That syntax `Self::fungible_id()` is not great since it gives the impression that all those smart contract have the same fungible_id.